### PR TITLE
Add Helper Card to Employees page

### PR DESCRIPTION
**Pull request by sarjon**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Adds helper card to employees page.
| Type?         | new feature
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | https://github.com/PrestaShop/PrestaShop/issues/10502#issuecomment-427389934
| How to test?  | Access `admin-dev/index.php/configure/advanced/employees` and you should see helper card at the very top.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11049)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/config/services/core/util/helper_card.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  prestashop.core.util.helper_card.team_helper_card_documentation_link_provider:
+    class: 'PrestaShop\PrestaShop\Core\Util\HelperCard\HelperCardDocumentationLinkProvider'
+    arguments:
+      - '@=service("prestashop.adapter.legacy.context").getContext().language.iso_code'
+      -
+        'en': 'http://doc.prestashop.com/display/PS17/Team'
+        'fr': 'http://doc.prestashop.com/pages/viewpage.action?pageId=51840334'
+        'es': 'http://doc.prestashop.com/display/PS17/Equipo'
+        'it': 'http://doc.prestashop.com/pages/viewpage.action?pageId=54266520'
+      - 'http://doc.prestashop.com/display/PS17/Team'
```
Comment (sarjon) [inline src/PrestaShopBundle/Resources/config/services/core/util/helper_card.yml]: im thinking about configuration for all card, like so:

```yaml
 arguments:
      - '@=service("prestashop.adapter.legacy.context").getContext().language.iso_code'
      -
       team:
          -        
            'en': 'http://doc.prestashop.com/display/PS17/Team'
            'fr': 'http://doc.prestashop.com/pages/viewpage.action?pageId=51840334'
            'es': 'http://doc.prestashop.com/display/PS17/Equipo'
            'it': 'http://doc.prestashop.com/pages/viewpage.action?pageId=54266520'
          - 'http://doc.prestashop.com/display/PS17/Team'
        meta:
          -        
            'en': 'http://doc.prestashop.com/display/PS17/SEO+and+URLs'
            'fr': 'http://doc.prestashop.com/display/PS17/SEO+et+URL'
            'es': 'http://doc.prestashop.com/display/PS17/SEO+y+URLs'
            'it': 'http://doc.prestashop.com/display/PS17/SEO+e+URL'
          - 'http://doc.prestashop.com/display/PS17/SEO+and+URLs'
```

and it the code `$helperCardDocumentationLinkProvider->getLink('team');` @matks wdyt?

**Comment by sarjon** [comment]
ready for review. :)

File: src/PrestaShopBundle/Resources/config/services/core/util/helper_card.yml (hunk 1 above)
Comment (mickaelandrieu) [reply src/PrestaShopBundle/Resources/config/services/core/util/helper_card.yml]: go for it :+1:

File: src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Employee/Blocks/helper_card.html.twig [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<div class="helper-card card">
+  <div class="helper-card__left helper-card__team shape-one">
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Employee/Blocks/helper_card.html.twig:27]: I don't really like the mix between " - " and " __ ": please chose one and stick with it please.

**Review by mickaelandrieu — DISMISSED** [review]
Minor changes asked, the ping me so I can't add the Waiting for QA label.

File: src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Employee/Blocks/helper_card.html.twig (hunk 2 above)
Comment (sarjon) [reply src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Employee/Blocks/helper_card.html.twig:27]: actually i didnt choose this style, i just reused it. :/ i believe this is some CSS coding style, but im not sure.

File: src/PrestaShopBundle/Resources/config/services/core/util/helper_card.yml (hunk 1 above)
Comment (matks) [reply src/PrestaShopBundle/Resources/config/services/core/util/helper_card.yml]: +1

**Review by matks — DISMISSED** [review]

**Comment by matks** [comment]
@sarjon Code is fine 👍 but if you want to apply https://github.com/PrestaShop/PrestaShop/pull/11049/files#r225888094 it's even better

**Comment by sarjon** [comment]
@matks updated it with https://github.com/PrestaShop/PrestaShop/pull/11049/files#r225888094 can you do a quick review again? :)

**Review by matks — DISMISSED** [review]

**Comment by matks** [comment]
@sarjon QA has validated, would you please rebase :) ?

**Comment by sarjon** [comment]
done. :+1:

**Review by PierreRambaud — APPROVED** [review]

**Comment by matks** [comment]
Thanks @sarjon

File: admin-dev/themes/new-theme/js/components/helper-card.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+const $ = window.$;
+
+/**
+ * Manages Helper cards on page.
+ */
+export default class HelperCard {
+  constructor() {
+    $(document).on('click', '.js-close-helper-card', (event) => {
+      console.log($(event.target));
```
Comment (matks) [inline admin-dev/themes/new-theme/js/components/helper-card.js:34]: @sarjon I just saw this. Is it intended or is it a debug statement that we forgot to remove ?

Comment (sarjon) [reply admin-dev/themes/new-theme/js/components/helper-card.js:34]: definitely not intended, we should remove it.

**Pull request merged**